source/unread/index.rst

```diff
@@ -0,0 +1,92 @@
+
+Install the plugin
+------------------
+
+-  Go to the marketplace. Download and install the plugin **Unread messages**.
+
+.. figure:: images/Unread-1.png
+   :alt:
+
+Setting up the plugin
+---------------------
+
+-  Go to **Administration > Entities**, **unread messages** tab
+
+There are 3 possible settings:
+
++----------------------------------------+-----------------------------+
+| Operation                              | Effect                      |
++========================================+=============================+
+| **Disabled**                           | The plugin will remain      |
+|                                        | inactive                    |
++----------------------------------------+-----------------------------+
+| **Per user helpdesk;share central**    | Users with the self-service |
+|                                        | profile will be notified of |
+|                                        | tickets for which they are  |
+|                                        | the requestor. Technicians, |
+|                                        | for their part, will share  |
+|                                        | the notifications. If a     |
+|                                        | notification is read, it    |
+|                                        | will be marked as read for  |
+|                                        | all other technicians       |
++----------------------------------------+-----------------------------+
+| **Per user**                           | Each user will have access  |
+|                                        | to their own queue of       |
+|                                        | notifications, including    |
+|                                        | those for their group and   |
+|                                        | their individual            |
+|                                        | assignments                 |
++----------------------------------------+-----------------------------+
+
+Display notifications
+---------------------
+
+Header
+~~~~~~
+
+When a new activity has not yet been viewed, a :bell::point-filled: sign appears:
+
+.. figure:: images/Unread-2.png
+   :alt:
+
+Listed
+~~~~~~
+
+Unread messages on a ticket highlight the ticket concerned and a red
+indicator :
+
+.. figure:: images/Unread-3.png
+   :alt:
+
+Ticket timeline
+~~~~~~~~~~~~~~~
+
+New messages will be identified by a banner **new messages**.
+
+.. figure:: images/Unread-4.png
+   :alt:
+
+Management by entity
+~~~~~~~~~~~~~~~~~~~~
+
+Management by entity is possible to manage the counting of notifications by entity.
+
+- Go to **administration > entities**, and select the desired entity.
+- In the **unread messages** tab, select the option that suits you so that unread messages behave differently to the parent entity.
```

Comment:
```suggestion
- Go to **Administration > Entities**, and select the desired entity.
- In the **Unread messages** tab, select the option that suits you so that unread messages behave differently to the parent entity.
```